framework/scenario/src/scenario_go_runner.rs

```diff
@@ -10,7 +10,7 @@ pub fn run_go<P: AsRef<Path>>(relative_path: P) {
     let mut absolute_path = std::env::current_dir().unwrap();
     absolute_path.push(relative_path);
 
-    let output = Command::new("mandos-test")
+    let output = Command::new("run-scenarios")
```

Comment:
Let's make a separate function with this flag, run_go_v2, for backwards compatibility.

I will refactor the whole thing early next week.

Reply:
I will also provide a proper test generator for the files where this is called.

Reply:
Or, another option: check that the binary exists before calling, and fallback to the old one if not.

Reply:
I added a fallback for this, which tries to run `run-scenarios` and if that fails with a `not found` error, it tries the old binary.